recipes/_install-windows.rb

```diff
@@ -141,11 +154,3 @@
     unsafe
   end
 end
-
-windows_env 'DDAGENTUSER_NAME' do
-  action :delete
-end
-
-windows_env 'DDAGENTUSER_PASSWORD' do
-  action :delete
-end
```

Comment:
I think we can still keep these deletes, so the env vars are cleaned earlier than at `run_completed` or `run_failed`. 

Eg: if a run takes long and you kill it, `on_failed` might not get called, so it's better to delete the env vars earlier when we know we don't need them anymore.